axelar-chains-config/src/utils/verifyContract.js

```diff
@@ -27,7 +27,12 @@ const verifyContract = (env, chain, contract, args, options = {}) => {
     console.log(`Verifying contract ${contract} with args '${stringArgs.join(',')}'`);
     console.log(cmd);
 
-    execSync(cmd, { stdio: 'inherit' });
+    try {
+        execSync(cmd, { stdio: 'inherit' });
```

Comment:
we already catch the error per chain in the verify script. what's the reason for this? If there's a failure for verification for a given chain, we'll have to rerun that anyways?

Reply:
> If for a new deployment there are 5 contracts that will be verified and if verifying the 1st one fails then all other 4 contracts too will not be verified.
This is not as big of a concern since you have to rerun it for that chain again anyways if there's any failures. I'm generally not a fan of hiding errors implicitly, and allow the caller to determine.

> From a list of 5 contracts if lets say 3rd one is upgraded and needs to be verified again. Then running the script should verify 3rd one and other 4 should say as already verified. But for fantom tetsnet the other contracts which are already verified throws error.
I see. Can we make the verify method more robust then? `verifyContract` should only fail on actual errors, not if the contract is already verified, so it needs to better handle the scenario for Fantom (e.g. can we check the error string, or can we check verification status explicitly first if chain is fantom), instead of ignoring all failures.